pkg/models/apikey.go

```diff
@@ -64,7 +68,8 @@ type GetApiKeyByIdQuery struct {
 // DTO & Projections
 
 type ApiKeyDTO struct {
-	Id   int64    `json:"id"`
-	Name string   `json:"name"`
-	Role RoleType `json:"role"`
+	Id         int64      `json:"id"`
+	Name       string     `json:"name"`
+	Role       RoleType   `json:"role"`
+	Expiration *time.Time `json:"expiration"`
```

Comment:
Use the following to omit empty fields being returned as null in the api response:

```
Expiration *time.Time `json:"expiration,omitempty"`
```

Reply:
GTK